js/src/lib/interfaces/Libraries.ts

```diff
@@ -610,12 +642,24 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: Partial<Record<ModelLibraryKey, Librar
 		repoUrl:  "https://github.com/NVIDIA/NeMo",
 		snippets: nemo,
 	},
+	"onnx": {
+		btnLabel: "ONNX",
```

Comment:
Should be `Optimum` IMO

![image](https://github.com/huggingface/hub-docs/assets/33657802/99456c87-4988-445f-b967-ebd813cb224a)